content/home/examples/a-component-using-external-plugins.js

```diff
@@ -2,7 +2,7 @@ class MarkdownEditor extends React.Component {
   constructor(props) {
     super(props);
     this.handleChange = this.handleChange.bind(this);
-    this.state = { value: 'Hello, **world**!' };
+    this.state = { value: 'Hallo, **Welt**!' };
```

Comment:
```suggestion
    this.state = { value: 'Hallo **Welt**!' };
```
I'd remove the comma, because it looks a bit "translated", what do you think?